### ISSUE TITLE
Clarification for the cv_results best method

### ISSUE THREAD

**Issue by GeeseAndQuack** [predictive-analytics-lab/EthicML#621]
Hi,

The best method says it  'Returns a model initialised with the hyper-parameters that perform optimally on average across folds for a given metric.'.  

In some cases for a given fairness metric the maximum value indicates poor performance of a metric, as such calling best for certain measures returns the worst performing model, because I believe the best call returns the max value in the mean_storage?

Could an argument to somehow specify what constitutes best for the given measure be added?

Thanks, and let me know if I am just plain wrong.

**Comment by olliethomas** [comment]
Good idea. We'd take a pull request with this feature if you'd like to work on it.

**Comment by thomkeh** [comment]
A work-around would be to define a new metric that is the negative of the existing metric.

```python
@dataclass
class InverseAccuracy(Accuracy):
    def score(self, prediction: Prediction, actual: DataTuple) -> float:
        return -super().score(prediction, actual)
```
